app/src/main/java/org/fossasia/openevent/app/module/auth/login/LoginFragment.java

```diff
@@ -114,26 +89,11 @@ public void onStop() {
 
     private void openSignUpPage() {
         getFragmentManager().beginTransaction()
-            .replace(R.id.fragmentContainer, new SignUpFragment())
+            .setCustomAnimations(R.anim.enter_from_left, R.anim.exit_from_right)
+            .replace(R.id.fragment_container, new SignUpFragment())
             .commit();
     }
 
-    private void setEditTextListener() {
-        binding.etBaseUrl.setOnEditorActionListener(this);
-        binding.emailDropdown.setOnEditorActionListener(this);
-        binding.etPassword.setOnEditorActionListener(this);
-    }
-
-    @Override
-    public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
```

Comment:
Go button should only be added on the last field of the form, or else the next buttons won't work. And data binding should be used for it